gateway/1.4/modules/ROOT/pages/flex-architecture-basic-deployments.adoc

```diff
@@ -0,0 +1,41 @@
+= Basic Deployment Models for Flex Gateway
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Flex Gateway supports three basic deployment models, which apply to both Connected Mode and Local Mode:
+
+* <<standalone-deployment, Standalone Deployment>>
```

Comment:
"Deployment" should be lowercase in these entries because they are in context. (I can explain to you verbally if needed.)